blueocean-dashboard/src/main/js/creation/bitbucket/cloud/BbCloudFlowManager.js

```diff
@@ -302,8 +302,21 @@ export default class BbCloudFlowManager extends FlowManager {
     _createPipelineComplete(result) {
         this.outcome = result.outcome;
         if (result.outcome === CreateMbpOutcome.SUCCESS) {
-            this.changeState(STATE.STEP_COMPLETE_SUCCESS);
-            this.pipeline = result.pipeline;
+            if (!this.isStateAdded(STATE.STEP_COMPLETE_MISSING_JENKINSFILE)) {
+                setTimeout(() => this._checkForBranchCreation(result.pipeline.name, true, ({ isFound, hasError, pipeline }) => {
```

Comment:
`_checkForBranchCreation` already has a configurable `delay` param, so wrapping this in another timeout may be unnecessary?